### PR TITLE
feat: add videos tab

### PR DESCRIPTION
**Pull request by edmonday**
# Description

### Issue

A videos tab needed to be added to the navbar.

[Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/37071156/todos/7336692389)

### Solution

Grouped existing links under a "Youtube" nested list. Added a link to Videos. Renamed several components.

# External Changes

_If you have made changes to external services, need to add additional values to Doppler, or need to add something new to the database, explain it here. This may include updates to third-party services, changes to infrastructure configuration, integration with external APIs, etc._

# Additional information

_Provide any additional information that might be useful to the reviewer in evaluating this pull request. This could include performance considerations,design choices, etc._

### CODE REVIEW

File: apps/nexus-admin/src/components/PageWrapper/Header/Header.tsx [hunk 1]
```diff
@@ -10,19 +10,19 @@ import MenuItem from '@mui/material/MenuItem'
...
 import { FC, MouseEvent, useState } from 'react'
 
 interface HeaderProps {
   title?: string
   hasBack?: boolean
+  user?: User
 }
 
-export const Header: FC<HeaderProps> = ({ title, hasBack }) => {
+export const Header: FC<HeaderProps> = ({ title, hasBack, user }) => {
```
Comment (Kneesal) [inline apps/nexus-admin/src/components/PageWrapper/Header/Header.tsx:23]:
```suggestion
export function Header ({ title, hasBack, user }): ReactElement  {
```

File: apps/nexus-admin/src/components/PageWrapper/Header/Header.tsx [hunk 2]
```diff
@@ -35,11 +35,16 @@ export const Header: FC<HeaderProps> = ({ title, hasBack }) => {
...
+    await user?.signOut()
   }
 
   return (
-    <Stack direction="row" alignItems="center" justifyContent="space-between">
+    <Stack
+      direction="row"
+      alignItems="center"
+      justifyContent="space-between"
+      data-testid="main-header"
```
Comment (Kneesal) [inline apps/nexus-admin/src/components/PageWrapper/Header/Header.tsx:46]: change all data-testids in the files you have touched to pascal case

File: apps/nexus-admin/src/components/PageWrapper/NavigationDrawer/ListGroup/ListGroup.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  it('should show children in list group', () => {
+    const { getByTestId } = render(
+      <ListGroup name="group">
+        <div>Child</div>
+      </ListGroup>
+    )
+    fireEvent.click(getByTestId('ListGroupToggle'))
+    expect(getByTestId('Collapse')).toHaveTextContent('Child')
+  })
+})
```
Comment (Kneesal) [inline apps/nexus-admin/src/components/PageWrapper/NavigationDrawer/ListGroup/ListGroup.spec.tsx:15]: write proper test

File: apps/nexus-admin/src/components/PageWrapper/NavigationDrawer/ListGroup/ListGroup.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import Collapse from '@mui/material/Collapse'
```
Comment (Kneesal) [inline apps/nexus-admin/src/components/PageWrapper/NavigationDrawer/ListGroup/ListGroup.tsx:1]: Story for this file?

File: apps/nexus-admin/src/components/PageWrapper/NavigationDrawer/NavigationDrawer.tsx [hunk 5]
```diff
@@ -0,0 +1,196 @@
+import CampaignIcon from '@mui/icons-material/Campaign'
```
Comment (Kneesal) [inline apps/nexus-admin/src/components/PageWrapper/NavigationDrawer/NavigationDrawer.tsx:1]: Thoughts on using existing nav drawer to libs folder and extending its functionality so we can pass our own drawer buttons and then reusing it here?

File: apps/nexus-admin/src/components/PageWrapper/PageWrapper.tsx [hunk 6]
```diff
@@ -0,0 +1,73 @@
+import Stack from '@mui/material/Stack'
```
Comment (Kneesal) [inline apps/nexus-admin/src/components/PageWrapper/PageWrapper.tsx:1]: I wonder if we will be able to move PageWrapper in journeys-admin to shared/libs and reuse that here as well? Can you please look into this?

**Review by Kneesal — COMMENTED** [review]
On second thought. Would be good to split this PR into two. 
One PR is all your storybook configs/setups 
Second PR for moving common components to libs and reusing them 
thoughts?

File: apps/nexus-admin/.eslintrc.json [hunk 7]
```diff
@@ -28,7 +28,8 @@
             "allowAsStatement": true
           }
         ],
-        "@typescript-eslint/no-misused-promises": "off"
+        "@typescript-eslint/no-misused-promises": "off",
+        "i18next/no-literal-string": "off"
```
Comment (tataihono) [inline apps/nexus-admin/.eslintrc.json:32]: no sir. please fix the issues.

File: apps/nexus-admin/src/components/PageWrapper/Header/Header.tsx (hunk 2 above)
Comment (tataihono) [reply apps/nexus-admin/src/components/PageWrapper/Header/Header.tsx:46]: this is not right. the value is data-testid.

**Review by tataihono — CHANGES_REQUESTED** [review]

File: apps/nexus-admin/src/components/PageWrapper/Header/Header.tsx (hunk 2 above)
Comment (Kneesal) [reply apps/nexus-admin/src/components/PageWrapper/Header/Header.tsx:46]: ^ @tataihono examples of where we have moved to Pascal Case 
https://github.com/JesusFilm/core/pull/2014

**Pull request closed without merging**